src/content/developers/docs/intro-to-ether/index.md

```diff
@@ -0,0 +1,91 @@
+
+## Pricing a Transaction in Ether {#pricing-a-transaction}
+
+The price of a transaction on Ethereum is determined by two variables: gas and gas fees. By multiplying these variables, users can determine the price of their transaction — which is paid in Ether.
+
+[Gas](/developers/docs/gas/#what-is-gas) is proportional to the number of operations a given transaction must execute before completing. Some transactions might be very simple; for example, moving Ether from one account to another. Other transactions are much more intensive; for example, calculating the first trillion Fibonacci numbers. Tracking gas ensures each user pays a proportional amount for their usage of the Ethereum network's computing power.
+
+Gas fees are a measure of the short-term demand for the Ethereum network's computing power. When many users are trying to use Ethereum, gas fees increase (and vice versa).
+
+Gas fees are made up of two components: [base fee](/developers/docs/gas/#base-fee) and [priority fee](/developers/docs/gas/#priority-fee) (also called a tip). The base fee is set by the protocol in reaction to network demand, and the priority fee is set by users hoping to be included into the next block. Senders can set their priority fee as high as they prefer; however, the miners who include transactions into blocks are incentivized to pick transactions offering the highest gas fees.
+
+Thus, gas fees functions as an auction mechanism for inclusion in the next Ethereum block. As a result, two transactions with the same amount of gas may cost more or less depending on when they are submitted to Ethereum.
+
+[More on gas](/developers/docs/gas/)
+
+## Minting Ether {#minting-ether}
+
+Ether is minted by the Ethereum protocol when a miner creates a block. To incentivize block creation, the protocol grants a "reward" in each block, incrementing the balance of an address set by the block's miner. The block reward has changed over time, and today it is 2 ETH per block.
+
+Ethereum also provides smaller rewards to miners who create ["uncle blocks"](/developers/docs/data-and-analytics/block-explorers/#blocks), which are beyond the scope of this introduction.
+
+## Burning Ether {#burning-ether}
+
+Ether burn occurs in every transaction on Ethereum. When a user pays for their transaction, their [base fee](/developers/docs/gas/#base-fee) is automatically destroyed. [In some blocks](https://etherscan.io/block/12965263), more Ether are burned than minted due to base fee burn.
+
+## Denominations of Ether {#denominations}
+
+Since many transactions on Ethereum are small, Ether has several denominations which may be referenced for smaller amounts.
+
+The denominations below can be found in the [Ethereum Whitepaper](/whitepaper/#currency-and-issuance). Others have been proposed, but Wei, Szabo, and Finney remain the most commonly used denominations. Each is named after a famous member of the cypherpunks, a group of cryptographers whose work influenced today's blockchain space.
```

Comment:
> Others have been proposed, but Wei, Szabo, and Finney remain the most commonly used denominations.

Isn't gwei used more commonly than all of these?

Reply:
Good point. Will rearrange this section

Reply:
Ended up removing denominations outside of Wei/Gwei. Szabo/Finney felt more like fun facts rather than directly useful denominations (at least I haven't seen them used much). Happy to re-add if you feel differently